kafka_consumer/hatch.toml

```diff
@@ -4,7 +4,7 @@
 # If you bump the `confluent-kafka` version, also bump the `librdkafka` version in the `32_install_kerberos.sh` file
 post-install-commands = [
   "python -m pip uninstall -y confluent-kafka",
-  "python -m pip install --no-binary confluent-kafka confluent-kafka==2.8.0",
```

Comment:
We probably need to bring a newer one now that we are using confluent kafka and dropped bitnami?

Reply:
Note: I assumed that they might have a python client with versions matching the confluent platform version we are using (as is commonly done), but looking at the client versions this is not the case. 2.11.1 seems to be the latest one so we should be good there. It was a wild guess about why tests could be failing.